frontend/src/metabase/components/Input/Input.tsx

```diff
@@ -28,14 +34,12 @@ const Input = ({ error, fullWidth, helperText, ...rest }) => {
   );
 };
 
-const InputHelpContent = forwardRef(function InputHelpContent(props, ref) {
+const InputHelpContent = forwardRef(function InputHelpContent(props, ref: any) {
```

Comment:
With the current version of `styled-components` it's impossible not to use `any` for ref.